src/clients/bridges/BaseAdapter.ts

```diff
@@ -69,11 +83,16 @@ export class BaseAdapter {
     };
   }
 
-  getSearchConfig(chainId: number) {
+  getSearchConfig(chainId: number): {
+    fromBlock: number;
+    maxBlockLookBack?: number;
+    concurrency?: number;
+    toBlock?: number;
+  } {
```

Comment:
```suggestion
  getSearchConfig(chainId: number): EventSearchConfig {
```

Requires: `import { EventSearchConfig } from "../../utils";`